app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -194,20 +178,23 @@ public void onPairingButtonClicked(final View view) {
         final Intent intent;
         switch (view.getId()) {
             case R.id.pairing_menu_bluetooth:
-                intent = new Intent(this, ActivateBluetoothActivity.class);
+                this.bluetoothButtonPressed = true;
+                this.enableBluetooth();
```

Comment:
Bluetooth should be enabled as soon as the user logs in, but we'll have to talk about how we want do do that (eg ask the user if he wants that). Made an issues for that: #198